README.md

```diff
@@ -38,36 +38,47 @@ Once the dependencies are installed, run
 
 ## Executables
 
-Go Ethereum comes with several wrappers/executables found in 
-[the `cmd` directory](https://github.com/ethereum/go-ethereum/tree/develop/cmd):
+Go Ethereum comes with several wrappers/executables found in the `cmd` directory.
```

Comment:
[![Write in Go](https://img.youtube.com/vi/LJvEIjRBSDA/0.jpg)](https://www.youtube.com/watch?v=LJvEIjRBSDA)